.circleci/continue_config.yml

```diff
@@ -165,9 +165,6 @@ jobs:
       - run:
           name: Install envsubst
           command: sudo apt-get update && sudo apt-get install gettext-base
-      - run:
-          name: Install ODBC
-          command: sudo apt-get install unixodbc-dev
```

Comment:
We don't need these anymore?

Reply:
i assume we still need this for ci/cd tests right?

Reply:
The airflow tests don't include the ci/cd only tests, so the airflow-specific docker doesn't need the dependency